README.md

```diff
@@ -526,10 +527,49 @@ Supported options for INSTAGRAM_STORIES:
 | **SHARE_STICKER_IMAGE** | stickerImage   |
 | **SHARE_BACKGROUND_AND_STICKER_IMAGE** | backgroundImage, stickerImage   |
 
+### Static Values for Facebook Stories
+These can be assessed using Share.Social property
+For eg.
+```javascript
+import Share from 'react-native-share';
+
+const shareOptions = {
+    method: Share.FacebookStories.SHARE_BACKGROUND_AND_STICKER_IMAGE,
+    backgroundImage: 'http://urlto.png',
+    stickerImage: 'data:image/png;base64,<imageInBase64>', //or you can use "data:" link
```

Comment:
Do this works sharing with a URL?

Reply:
Yes it's working with both base64 data image and url.

Reply:
Maybe we can let implicit  that this param accept both `url` and `base64`.

Reply:
> Maybe we can let implicit that this param accept both `url` and `base64`.

Yes. So should me update the example source code on README file?

Reply:
Same here.